### PR TITLE
fix: fix authorization

### PR DESCRIPTION
**Pull request by flywukong**
### Description

The logic of checking the existence of the account has been removed from the VerifyAuthentication function which is used to verify authorization of the request to SP.  The reasons for the removal are as follows:：
1）Data marketplace users only operate on BSC and have no transactions on Greenfield.
2）During batch upload, a temporary account is generated, and the gas and fees for this temporary account are deducted from the user's account (with authorization). This temporary account has not yet been created on the chain, but actions like createObject are performed

### Rationale

support marketplace user case

### Example

NA

### Changes

Notable changes: 
*  remove checking the existence of the account

### CODE REVIEW

**Review by RenRick — APPROVED** [review]
LGTM

File: modular/gater/object_handler.go [hunk 1]
```diff
@@ -266,7 +267,7 @@ func (g *GateModular) getObjectHandler(w http.ResponseWriter, r *http.Request) {
 			enableCheck, reqCtx.Account(), uint64(highOffset-lowOffset+1), pInfo.SegmentPieceKey, pInfo.Offset,
 			pInfo.Length, g.baseApp.TaskTimeout(task, uint64(pieceTask.GetSize())), g.baseApp.TaskMaxRetry(task))
 		getSegmentTime := time.Now()
-		pieceData, err := g.baseApp.GfSpClient().GetPiece(reqCtx.Context(), pieceTask)
+		pieceData, err = g.baseApp.GfSpClient().GetPiece(reqCtx.Context(), pieceTask)
```
Comment (Fynnss) [inline modular/gater/object_handler.go:270]: Please remove those irrelevant modifications

Comment (flywukong) [reply modular/gater/object_handler.go]: removed

**Review by sysvm — APPROVED** [review]
LGTM

**Review by Fynnss — APPROVED** [review]
LGTM

**Pull request merged**